verilog/CST/declaration.cc

```diff
@@ -147,6 +147,19 @@ const verible::TokenInfo& GetTypeTokenInfoFromDataDeclaration(
   return instance_symbol_identifier->get();
 }
 
+const verible::SyntaxTreeNode* GetParamListFromDataDeclaration(
```

Comment:
> Can you provide more illustration?

Reply:
Sorry, I should have been clearer.  
Getting type parameters from a type parameter seems more re-usable than getting it from a full data declaration; types can appear other places (such as being passed as type parameters). 
So if you split/reduce this functionality, you'll be able to cover more in the future (with less work).

Reply:
I don't get the full context here because I don't have the full picture about parameter declarations yet.
But I will leave at TODO for this in case I used this function again then I can see how to make it re-usable. 
Is this okay?

Reply:
OK